flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/MySqlTestUtils.java

```diff
@@ -59,6 +59,7 @@ public static MySqlSource.Builder<SourceRecord> basicSourceBuilder(
                 .username(database.getUsername())
                 .password(database.getPassword())
                 .deserializer(new ForwardDeserializeSchema())
+                .serverTimeZone("UTC")
```

Comment:
minor: only `testValidateTimezone` need this parameter, but we set it as a fixed value for all legacy tests.

Reply:
Hi @leonardBang , the reason for setting `serverTimezone` to UTC is that the containerized MySQL for testing runs in UTC timezone by default, however, the test cases might be run on computers at different timezones, which could eventually cause test failures, and this is actually desired as we need to throw an exception when the local timezone is different from the server and no serverTimeZone is set by the user.

One possible solution is to dynamically append the local `default-time-zone`  settings to `test/resources/docker/server/my.cnf` and `docker/server-gtids/my.cnf`, so that the containerized MySQL runs in the local timezone and the test cases pass.

Another solution is to manually set the `serverTimezone` to UTC, so that the source has the same timezone as the server. As we have already tested timezone auto-detection cases in 'MySqlTableSourceFactoryTest.java', personally I think this approach is simpler.

Just my two cents, and looking for your reply :)

Reply:
I mean MySqlTestUtils#basicSourceBuilder method should has a `serverTimezone` parameter which is more flexible

Reply:
Hi @leonardBang , I have added a `serverTimezone` parameter in this method as recommended. However, AFAIK when running test suites in non-UTC timezones, most of the existing MySQL test cases need to be explicitly set to UTC timezone in accordance with the MySQL server timezone settings, so the actual argument for this method should still be 'UTC' : )